File: app/src/main/java/com/widyatama/nurseassistant/features/splash/SplashActivity.java
package com.widyatama.nurseassistant.features.splash;

import android.os.Bundle;
import android.os.Handler;

import com.widyatama.core.base.BaseActivity;
import com.widyatama.nurseassistant.R;
import com.widyatama.nurseassistant.features.login.LoginActivity;

public class SplashActivity extends BaseActivity {

    @Override
    protected void onInitializedView(Bundle savedInstanceState) {
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                launchActivityClearAllStack(LoginActivity.class);
            }
        }, 3000);
    }

    @Override
    protected Integer setLayout() {
        return R.layout.activity_splash;
    }
}

File: app/src/main/java/com/widyatama/nurseassistant/features/splash/SplashActivity.kt
package com.widyatama.nurseassistant.features.splash

import android.os.Bundle
import android.os.Handler
import com.widyatama.core.base.BaseActivity
import com.widyatama.nurseassistant.R
import com.widyatama.nurseassistant.features.login.LoginActivity

class SplashActivity : BaseActivity() {

    override fun onInitializedView(savedInstanceState: Bundle?) {
        Handler().postDelayed({
            launchActivityClearAllStack(LoginActivity::class.java)
        },3000)
    }

    override fun setLayout(): Int {
        return R.layout.activity_splash
    }
}
